frigate/detectors/plugins/rknn.py

```diff
@@ -85,6 +85,9 @@ def parse_model_input(self, model_path, soc):
         if "/" in model_path:
             model_props["preset"] = False
             model_props["path"] = model_path
+            for model_type in ModelTypeEnum:
```

Comment:
I don't think this makes sense, users still need to provide the model type, we should just make sure it is set to what the user set

Reply:
make sense. Will close

Reply:
Not sure you needed to close, or is there no bug when the model type is set in the config?

Reply:
I tested by specifying the `model_type` in the config, and it works.